libraries/AP_HAL_ChibiOS/hwdef/CubeOrangePlus-ODID/hwdef.dat

```diff
@@ -0,0 +1,105 @@
+# hw definition file for processing by chibios_hwdef.py
+
+# MCU class and specific type
+MCU STM32H7xx STM32H757xx
+
+define CORE_CM7
+define SMPS_PWR
+
+include ../CubeOrange/hwdef.inc
```

Comment:
Why is this not including `CubeOrangePlus/hwdef.dat`?

Reply:
I did not change this as this is how the original CubeOrangePlus hwdef file was formatted. This section is identical to the CubeOrangePlus hwdef aside from the updated board id. Does this need to be changed?

Reply:
I have tested this to include CubeOrangePlus/hwdef.dat. These changes cause an error when configuring the board due to PF14 redefinition. I have not changed this file in the PR as it does not seem to reflect the original CubeOrangePlus build configurations. Please let me know if you have further questions or requests for this PR.

Reply:
So the concept is that you include `CubeOrangePlus/hwdef.dat` and put in just what you need to to do ODID.

Check out how `CubeOrange-ODID` does it.

Reply:
@BryonOI you still haven't addressed this comment.

The entirety of this file can simply be:
```
# hw definition file for processing by chibios_hwdef.py

include ../CubeOrangePlus/hwdef.dat

undef APJ_BOARD_ID
APJ_BOARD_ID 11063

define AP_OPENDRONEID_ENABLED 1
```

Similarly for the -bl file.